### ISSUE TITLE
Build CVE package page

### ISSUE THREAD

**Issue by steverydz** [canonical-web-and-design/ubuntu.com#7055]

**Comment by matthewpaulthomas** [comment]
I don’t think this issue is valid. As I wrote [in the design spec](https://docs.google.com/document/d/1VuAElUpFay7MBFNmE3NhSyY9TkXiqHcA1ZHQ11AYO-Q/edit#heading=h.du10b3k3teio), “There should be no dedicated package page (unlike the previous CVE tracker), because you can search for all CVEs in a particular package instead”.

It might make sense to implement a package page if package search can’t be implemented in time, but a dedicated package page could be. I’d be surprised if that’s true.

**Comment by steverydz** [comment]
Makes sense. I'll close it.
